client/assets/icons/BellIcon.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
```

Comment:
I really don't think this should be a react component, an `.scss` file would suffice.

Reply:
a `.svg` file you mean?

It might be a react component for two reasons:

- it's going to be reused across the project
- the icon has two states, the active and inactive state so if we make use of an `svg` then we'd need to create two separate files, but as a react component we can just be toggling the state.

What do you think?

Reply:
@williamolojede @BolajiOlajide I actually used this because the current webpack config we have doesn't allow us to use `.svg`. Will create a chore for it. But also it makes sense if it's an icon that is more likely going to have states in the future.

Reply:
What's the status on this PR?

Reply:
@andela-sjames I am the one working on this PR, I will resolve the conflicts and sort out some padding by EOD